tests/unit/s2n_security_policies_test.c

```diff
@@ -274,46 +274,34 @@ int main(int argc, char **argv)
     {
         struct s2n_config *config = s2n_config_new();
 
-        EXPECT_SUCCESS(s2n_config_set_cipher_preferences(config, "default"));
+        EXPECT_SUCCESS(s2n_config_set_security_policy(config, S2N_SECURITY_POLICY_DEFAULT));
```

Comment:
You should also duplicate these tests so that we're still testing s2n_config_set_cipher_preferences too.